### DISCUSSION TITLE
How i can execute a datalayer when user creates account

### DISCUSSION THREAD

**Discussion by b4d4** [PrestaShop/PrestaShop#33154] [Q&A]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

It's imposible to register datalayer when user creates successfuly account

### Steps to reproduce

Create account
Execute js code when successfuly account created

### PrestaShop version(s) where the bug happened

1.7.8

**Comment by kpodemski** [comment]
Hello @b4d4 

It's more like a question than a bug, so I'm transferring this issue to Discussions :)
